lib/python/treadmill_aws/hostmanager.py

```diff
@@ -183,7 +183,7 @@ def delete_hosts(ec2_conn, ipa_client, hostnames):
             record_zone = '{2}.{1}.{0}.in-addr.arpa.'.format(
                 *arecord.split('.'))
```

Comment:
Format can fail too, if split fails to produce *exactly* 3 elements

```
>>> '{} {}'.format(1,2)
'1 2'
>>> '{} {}'.format(1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: tuple index out of range
```

Reply:
ipv4 addresses should conform to that format. If the key returned by AWS is empty, the KeyError exception should handle this. I can use the ipaddress module to load the record and confirm it is a valid address prior to further processing, but I feel this is an edge case.